ocs_ci/deployment/helpers/external_cluster_helpers.py

```diff
@@ -411,6 +411,19 @@ def remove_rbd_images(self, images, pool):
             cmd = f"rbd rm {each_image} -p {pool}"
             self.rhcs_conn.exec_cmd(cmd)
 
+    def enable_secure_connection_mode(self):
+        """
+        Enables secure connection mode for RHCS cluster
+        """
+        logger.debug("Enabling secure connection mode for external RHCS cluster")
+        cmds = (
+            "ceph config set global ms_client_mode secure;"
```

Comment:
Do we need to disable it in the teardown or doesn't matter?

Reply:
Its better to disable during teardown. Tested on UMB cluster and even though if we ran the same commands twice, it doesn't throw any error. I will do it as a followup PR.